dashboard/app/controllers/test_controller.rb

```diff
@@ -100,6 +100,47 @@ def create_script
     render json: {script_name: script.name, lesson_id: lesson.id}
   end
 
+  # Create a script containing a single lesson group, lesson and script level that has the is_migrated setting
+  def create_migrated_script
```

Comment:
really nice additions to the test code here! I think this looks like the right new code, but I am wondering if it makes sense to remove everything related to lesson activities and activity sections from `create_script` above. that way, our tests are testing the same kind of scripts as the ones that are actually getting edited with the legacy script editor today. you would pretty much be undoing this commit: https://github.com/code-dot-org/code-dot-org/commit/43952d6c3c584464de05c340bcc51eccb19c5ecb if the tests still pass after doing that, then that's what I'd recommend.